src/UI/Seller/src/app/config/app.config.ts

```diff
@@ -9,9 +9,11 @@ export const ocAppConfig: AppConfig = {
   appname: 'OrderCloud Admin',
   clientID: environment.clientID,
   middlewareUrl: environment.middlewareUrl,
+  buyerID: environment.buyerID,
```

Comment:
you can get buyer i do with a Me.Get, this could be in appstateservice though

Reply:
Me.Get won't return a BuyerId if I'm logged in as an admin, right?